workflows/foobernetes_pp.pp

```diff
@@ -17,10 +17,12 @@
     loadBalancerIP => '10.0.0.1',
     location => 'eu1',
     replica => false,
+    policy => $load_balancer_policy,
     webServerIDs => [$webserver1_id, $webserver2_id],
     tags => {
         'team' => 'lyra team',
-        'role' => 'primary'
+        'role' => 'primary',
+        'tags' => $load_balancer_policy
```

Comment:
I suggest removing the policy tag everywhere now when a proper "policy" attribute exist.